index.md

```diff
@@ -1,7 +1,83 @@
-# Welcome to `nmstate`
+# We are nmstate!
+A descriptive network manager API for hosts.
```

Comment:
what do you mean by "descriptive"? it is not clear to me. Maybe you'd like to sat "declarative", as it declares the requested (and current) state, not the procedure to get to it.